src/Symfony/Component/Workflow/Registry.php

```diff
@@ -24,17 +23,11 @@ class Registry
     private $workflows = array();
 
     /**
-     * @param Workflow                        $workflow
-     * @param string|SupportStrategyInterface $supportStrategy
+     * @param Workflow                 $workflow
+     * @param SupportStrategyInterface $supportStrategy
      */
-    public function add(Workflow $workflow, $supportStrategy)
+    public function add(Workflow $workflow, SupportStrategyInterface $supportStrategy)
```

Comment:
Didn't notice the already-present BC notice on 3.*, but still, it's still a BC break to make this change on 3.x